portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -100,11 +108,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       response.sendRedirect("/index.html");
       return;  
     }
-
+    
     Entity commentEntity = new Entity("Comment"); 
     commentEntity.setProperty("message", comment);
     commentEntity.setProperty("name", name);   
     commentEntity.setProperty("picture", pictureURL);
+    commentEntity.setProperty("posterId", key);
```

Comment:
Nit: I find it slightly confusing that key/id is used to refer to the same thing in different places.